armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -49,6 +56,12 @@ BASE_BITCOIN_NETWORK=${BASE_BITCOIN_NETWORK:-"testnet"}
 BASE_AUTOSETUP_SSD=${BASE_AUTOSETUP_SSD:-"false"}
 BASE_WIFI_SSID=${BASE_WIFI_SSID:-""}
 BASE_WIFI_PW=${BASE_WIFI_PW:-""}
+BASE_DASHBOARD_WEB_ENABLED=${BASE_DASHBOARD_WEB_ENABLED:-"false"}
+BASE_HDMI_BUILD=${BASE_HDMI_BUILD:-"true"}
+
+# HDMI dashboard only enabled if image is built to support it
+if [[ "${BASE_HDMI_BUILD}" != "true" ]]; then BASE_DASHBOARD_HDMI_ENABLED="false"; fi
```

Comment:
Nit: Could we use the same style for `if` statements as elsewhere? I.e:

```
if [[ "${BASE_HDMI_BUILD}" != "true" ]]; then
  BASE_DASHBOARD_HDMI_ENABLED="false"
fi
```

Reply:
Well, I'm really miserly with lines of code... :-) Fixed.